components/movementsensor/imuwit/imu.go

```diff
@@ -35,9 +38,31 @@ type AttrConfig struct {
 
 // Validate ensures all parts of the config are valid.
 func (cfg *AttrConfig) Validate(path string) error {
+	var isValid = false
+
+	// Validating serial path
 	if cfg.Port == "" {
 		return utils.NewConfigValidationFieldRequiredError(path, "serial_path")
 	}
+
+	// Validating baud rate
+	for _, val := range baudRateList {
+		if val == cfg.BaudRate {
+			isValid = true
+		}
+	}
+	if !isValid {
+
+		fmt.Print("\n")
+		fmt.Print("Please enter values from: ")
```

Comment:
instead of the `InvalidIntegerValue` function, could we make a specific `InvalidBaudRate` function in this file and have the print statements here be the message of the error returned by that function?